lib/ui/navbar.js

```diff
@@ -18,8 +21,14 @@ import {
   storeStartConnected,
   retrieveStartConnected,
   storeColorScheme,
-  retrieveColorScheme
+  retrieveColorScheme,
+  retrieveToken,
+  isPreferredUnits,
+  storePreferredUnits
 } from '../util/localstorage';
+import {
+  retrievePreferredUnits as ssRetrievePreferredUnits
```

Comment:
why the rename here?

Reply:
I've used prefix `ss` (ServerStorage) as opposed to `ls` (LocalStorage).
The functions are almost the same in localstorage & serverstorage, I prefix them when I have to use both in the same module.
I can replace by `ApplicationData` to speak the same language as the server.
As you want.